NetKAN/ClassicStockResources.netkan

```diff
@@ -1,23 +1,19 @@
-{
-    "spec_version": "v1.4",
-    "identifier":   "ClassicStockResources",
-    "author":       "Angel-125",
-    "$kref":        "#/ckan/github/Angel-125/ClassicStockResources",
-    "$vref":        "#/ckan/ksp-avc",
-    "license":      "GPL-3.0",
-    "resources": {
-        "homepage": "https://github.com/Angel-125/ClassicStockResources"
-    },
-    "tags": [
-        "library",
-        "resources"
-    ],
-    "depends": [
-        { "name": "ModuleManager"                  },
-        { "name": "WildBlue-PlayMode-ClassicStock" }
-    ],
-    "install": [ {
-        "find":       "ClassicStockResources",
-        "install_to": "GameData/WildBlueIndustries"
-    } ]
-}
+spec_version: v1.4
+identifier: ClassicStockResources
+author: Angel-125
+$kref: '#/ckan/github/Angel-125/ClassicStockResources'
+$vref: '#/ckan/ksp-avc'
+license: GPL-3.0
+resources:
+  homepage: https://github.com/Angel-125/ClassicStockResources
+tags:
+  - library
+  - resources
+conflicts:
+  - name: CommunityResourcePack
+depends:
+  - name: ModuleManager
+  - name: WildBlue-PlayMode-ClassicStock
```

Comment:
we want to remove this one right?

Reply:
I don't think so?

Reply:
I'm pretty sure Simplified play mode requires it

Reply:
Can you say why?

Reply:
Confirmed that the Simplified play mode disables ClassicStockResources, see https://github.com/KSP-CKAN/NetKAN/issues/9899#issuecomment-1892552659.

Reply:
I still think this is wrong, there are a number of mods that use ClassicStockResources but don't need a WBI play mode.

https://github.com/search?q=repo%3AKSP-CKAN%2FNetKAN%20ClassicStockResources&type=code

Reply:
ClassicStockResources is part of the ClassicStock play mode. If a mod "uses" (note, that's not the same as `supports` or `suggests`, which is most of what is linked there) it, then it uses the ClassicStock play mode.

Reply:
> ClassicStockResources is part of the ClassicStock play mode

I don't think this is true.  Other playmodes may disable ClassicStockResources, but that's not the same thing.

Reply:
Right. All but one .cfg file is part of that play mode. The remaining one is generically available. That's modeled in the latest changes. I'd recommend taking a look at the files, then opening CKAN, clicking Refresh, and taking it for a spin to see how it all works.